<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumePluginGUI</class>
 <widget class="QDialog" name="VolumePluginGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="global_RB">
       <property name="text">
        <string>Global</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="by_type_RB">
       <property name="text">
        <string>By Cell Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="by_cell_RB">
       <property name="text">
        <string>By Cell</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="by_type_GB">
     <property name="title">
      <string>Global</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_PB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_PB">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_PB</sender>
   <signal>clicked()</signal>
   <receiver>VolumePluginGUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_PB</sender>
   <signal>clicked()</signal>
   <receiver>VolumePluginGUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
